src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -143,8 +144,39 @@ function shouldShowDefaultBadge(filteredPaymentMethods, isDefault = false) {
 function isPaymentMethodActive(actionPaymentMethodType, activePaymentMethodID, paymentMethod) {
     return paymentMethod.accountType === actionPaymentMethodType && paymentMethod.methodID === activePaymentMethodID;
 }
-function PaymentMethodList(props) {
-    const {actionPaymentMethodType, activePaymentMethodID, bankAccountList, fundList, filterType, network, onPress, shouldShowSelectedState, selectedMethodID, translate} = props;
+
+/**
+ * @param {Object} item
+ * @returns {String}
+ */
+function keyExtractor(item) {
+    return item.key;
+}
+
+/**
+ * This is a part of the FlashList configuration. It is used to estimate the size of each item in the list.
+ * https://shopify.github.io/flash-list/docs/usage/#estimateditemsize
+ * Measured using Element Inspector: 64.
+ */
+const ESTIMATED_ITEM_SIZE = 64;
```

Comment:
![Simulator Screenshot - iPhone 15 - 2023-10-09 at 18 09 21](https://github.com/Expensify/App/assets/42047036/5121133e-69f4-4f87-aa8a-c4fb3acd6e59)

Reply:
This value is hard-coded [here](https://github.com/Expensify/App/blob/ce892f7efed6b6987daa65a4e604333ef482cc71/src/styles/styles.ts#L3163), please extract it to a constant in `src/styles/variables` and reference the constant in both places. Adjust the comment accordingly